src/screens/Connectors/ConnectorMetaData/ApplePay/ApplePaySimplifiedFlow.res

```diff
@@ -160,6 +161,15 @@ let make = (
         />,
       )
     />
+    <RenderIf condition={featureFlagDetails.isLiveMode && featureFlagDetails.complianceCertificate}>
+      <hr className="w-full" />
+      <SimplifiedHelper
+        heading="Get feature enabled from stripe"
+        subText=None
+        stepNumber="4"
+        customElement=Some(<SampleEmail />)
```

Comment:
Why it is in this way?
```
 <SimplifiedHelper
        heading="Get feature enabled from stripe"
        subText=None
        stepNumber="4"
        customElement=Some(<SampleEmail />)
        />
```
customElement - should have been optional field instead of value being optional. Can we please make this change?

Reply:
https://github.com/juspay/hyperswitch-control-center/pull/1063

Reply:
Raised just check - https://github.com/juspay/hyperswitch-control-center/pull/1063